src/call-session/index.ts

```diff
@@ -98,6 +127,7 @@ class CallSession extends EventEmitter {
       video: false,
       audio: { deviceId: { exact: this.inputDeviceId } },
     });
+    this.emit("userMedia", this.mediaStream);
```

Comment:
I am OK with this. But could you please clarify why do you need it? 
I tend to expose as less information as possible to make the SDK simpler.

Reply:
Do you think we should have the noise reduction feature build into the SDK?

Reply:
https://github.com/ringcentral/ringcentral-web-phone/commit/745990dc978b57c6b83c025dbac98b344fed4f19